R/plot_index.R

```diff
@@ -52,7 +52,7 @@ plot_index <- function(dir, recursive = TRUE, area = NULL, filter = NULL,
 
   #get the index information that is saved to the disk
   dir <- normalizePath(dir, mustWork = TRUE)
-  files <- dir(dir, pattern = "Table_for_SS3", full.names = TRUE,
+  files <- dir(dir, pattern = "Index", full.names = TRUE,
```

Comment:
Seems like there would be less changes if we just used "our" Table_for_SS3" rather than the automated Index.csv. That way only the csv would need to be saved and all downstream code would work b/c the csv would have the previously utilized column names. This is different than what I said in our previous chat, but I think rather than changing given that Jim doesn't use years that we should just use what we had before.